package.json

```diff
@@ -53,6 +50,7 @@
   "engines": {
     "node": ">=4"
   },
+  "files":["@hoodie/admin", "@hoodie/client", "@hoodie/server"],
```

Comment:
when you install `hoodie` like it’s described in the [quickstart guide](http://docs.hood.ie/en/latest/guides/quickstart.html) and then open `node_modules/hoodie`, you will see a ton of files in there, including the pandoc file which is 20mb of size. Hoodie doesn’t need that file to work, nor does it need e.g. the docs folder. The only file it needs are the JavaScript files that get executed when you run `npm start` in your app. These are `bin/`, `cli/` and `server/`. And then it also needs the `public/` folder as it has the default UI that will be shown when your app has no `own` public folder. So the names of these 4 folders are the one that we need to to set `"files"` to in our `package.json`. That way only these folders will be uploaded to the npm registry, all other files will be ignored. Does that make sense?